@here/harp-mapview/lib/MapView.ts

```diff
@@ -2956,7 +3002,7 @@ export class MapView extends THREE.EventDispatcher {
             this.m_camera.lookAt(this.scene.position);
         }
 
-        this.m_lookAtDistance = defaultGeoCenter.altitude!;
+        this.m_targetDistance = defaultGeoCenter.altitude!;
```

Comment:
I think the usage of `MapViewDefaults.geoCenter` is a bit inconsistent. Here it is used as camera position but above it is used to initialize the camera target:
```
    private m_targetGeoPos = new GeoCoordinates(
        MapViewDefaults.geoCenter.latitude,
        MapViewDefaults.geoCenter.longitude,
        0
    );
```

Reply:
Ok, will move this discussion to follow up ticket, where we will implement the target point and zoom level distance proper relation as also free camera mode.
See HARP-8371.